### PR TITLE
Include jsonnet-language-server installation

### PR DESCRIPTION
**Pull request by zalegrala**

### CODE REVIEW

**Comment by zalegrala** [comment]
How do I associate a filetype with this server?

**Comment by williamboman** [comment]
> How do I associate a filetype with this server?

This is done in lspconfig. I see you opened a PR there 👍

File: lua/nvim-lsp-installer/servers/init.lua [hunk 1]
```diff
@@ -14,11 +14,12 @@ local INSTALL_DIRS = {
...
     ["elixirls"] = "elixir",
     ["elmls"] = "elm",
-    ["eslint"] = "vscode-eslint",
     ["eslintls"] = "eslint",
+    ["eslint"] = "vscode-eslint",
     ["gopls"] = "go",
     ["hls"] = "haskell",
     ["intelephense"] = "php",
+    ["jsonnet"] = "go",
     ["kotlin_language_server"] = "kotlin",
```
Comment (williamboman) [inline lua/nvim-lsp-installer/servers/init.lua:23]: You can revert these diffs. No need to provide an entry for jsonnet here (see the comment above)

Comment (zalegrala) [reply lua/nvim-lsp-installer/servers/init.lua]: I've backed out the change to this file.

**Review by williamboman — APPROVED** [review]
Could you add it to the README as well as `lua/nvim-lsp-installer/servers/init.lua` (this file contains the index of all available servers, I think you reverted a bit too much 😄)?

Will wait for lspconfig PR to merge

**Comment by zalegrala** [comment]
I've made an update to the readme as well.  Let me know if any other changes need to be made.  Did any changes need to be made to the `lua/nvim-lsp-installer/servers/init.lua`?

**Comment by williamboman** [comment]
Yeah it's just a one-liner though, it needs to be added to `CORE_SERVERS`

**Review by williamboman — APPROVED** [review]

**Pull request merged**